### ISSUE TITLE
Is replace needed on mirror instructions?

### ISSUE THREAD

**Issue by danielcompton** [clojars/clojars-web#565]
The mirror instructions for leiningen recommend using `^:replace`. However that replaces any user defined repos as well in a `project.clj`.

with replace:

```
$ lein pprint :repositories
(:repositories detected in user-level profiles! [:user]
See https://github.com/technomancy/leiningen/wiki/Repeatability)
[["central" {:url "http://repo1.maven.org/maven2", :snapshots false}]
 ["clojars" {:url "https://beta.clojars.org/repo/"}]]
```

without:

```
$ lein pprint :repositories
(:repositories detected in user-level profiles! [:user]
See https://github.com/technomancy/leiningen/wiki/Repeatability)
(["central" {:url "http://repo1.maven.org/maven2", :snapshots false}]
 ["clojars" {:url "https://beta.clojars.org/repo/"}]
 ["releases"
  {:url "s3p://internal/releases/",
   :username :env/aws_access_key_id,
   :passphrase :env/aws_secret_access_key,
   :sign-releases false}]
 ["snapshots"
  {:url "s3p://internal/snapshots/",
   :username :env/day8_aws_access_key_id,
   :passphrase :env/day8_aws_secret_access_key}])
```

Lein seems to be happy replacing `clojars` with the beta one, as long as the names are the same. As a bonus, this means that you don't need to provide the central repo here either.

Are there any downsides to removing the `^:replace` instruction?

**Comment by tobias** [comment]
I went back through my notes and irc logs for the period where I developed those instructions, and it appears that I settled on using `^:replace` based on feedback from someone in IRC, but I don't know if I actually tested without it, so it may not be required. It may have also been required because of a bug in the version of lein I was using at the time - I know there have been some merging bugs.

But, if it's not required, I'm fine with an update to the instructions (and it simplifies the instructions we'll send out for beta testing the new server and the CDN, which is nice).
